CHANGELOG.next.asciidoc

```diff
@@ -184,3 +184,5 @@
 - Bump node.js version for heartbeat/synthetics to 16.15.0
 - Support scheduled actions and cancellation of pending actions. {issue}393[393] {pull}419[419]
 - Add `@metadata.input_id` and `@metadata.stream_id` when applying the inject stream processor {pull}527[527]
+- Redact sensitive information on diagnostics collect command. {issue}[241]
+- Redact sensitive information on diagnostics collect command. {issue}[241] {pull}[566]
```

Comment:
```suggestion
- Redact sensitive information on diagnostics collect command. {issue}[241] {pull}[566]
```

Reply:
@AndersonQ You have missed this one. I just commited it.